### ISSUE TITLE
NavMenu items use the same css class for parent and child

### ISSUE THREAD

**Issue by efonsecab** [enkodellc/blazorboilerplate#765]
In the MatNavMenu control, the parent and child menu items seem to be using the exact same class for selected, which makes it difficult to have only a sub-element marked/customized/highlighted.
There does not seem to be a logic for "active" element.

**Comment by enkodellc** [comment]
@efonsecab I believe that should be address to MudBlazor. But I would need more details or a screenshot to determine.

**Comment by efonsecab** [comment]
Hello @enkodellc, thanks for the quick reply , it's on a customer project so I cannot share screenshots of that one, I'll try to reproduce it on another project and let you know.

**Comment by enkodellc** [comment]
No, problem. I would say add your own class to the parent menu or look at doing some css selectors for childs. or .first().. nth().. Something along those lines.
